src/v1/mapping.ts

```diff
@@ -36,6 +36,21 @@ export const ID_MAP = {
   rose: 'Rose Dining Hall'
 };
 
+export const REV_ID_MAP = {
+  "Alice Cook House": 'cook',
+  "Cafe Jennie": "cafejennie",
+  'Carl Becker House': "becker",
+  'Jansens at Bethe House': "bethe",
+  'Keeton House': "keeton",
+  'Kosher': "west104",
+  'North Star Marketplace': "appel",
+  'Okenshields': "oakies",
+  'Olin Libe Cafe': "libe",
+  'RPME': "rpcc",
+  'Risley': "risley",
+  'Rose Dining Hall': "rose"
+};
+
```

Comment:
There might be a better way to do this rather than hardcoding this in:
```
function swapKeyValues(obj) {
    res = {}
    Object.keys(obj).foreach(key => res[obj[key]] = key)
    return res
}

export const REV_ID_MAP = swapKeyValues(ID_MAP)
```

Reply:
I actually didn't even end up using the reverse map, so should I remove it for now or add in the code you suggested in case we need it later?